packages/docusaurus/docusaurus.config.js

```diff
@@ -229,11 +229,15 @@ const config = {
             dropdownActiveClassDisabled: true,
             dropdownItemsAfter: [
               {
-                label: `3.6.4`,
+                label: `3.8.1`,
                 href: `https://v3.yarnpkg.com`,
               },
               {
-                label: `1.22.19`,
+                label: `2.4.3`,
+                href: `https://v2.yarnpkg.com`,
+              },
```

Comment:
There isn't a separate v2 docs. https://v2.yarnpkg.com is the v3 docs

Reply:
Shall we still include the v2 versions?

Reply:
I don't think so, given it is not an accurate documentation for v2

Reply:
I have applied your suggestion for the documentation.

However, having https://v2.yarnpkg.com pointing to https://v3.yarnpkg.com is really confusing for the user. Shall we consider to host the documentation for v2 users somewhere while there are still some users using v2?